src/lib/helpers.ts

```diff
@@ -9,5 +11,29 @@ export function uuid() {
 }
 
 export function stripTrailingSlash(url: string) {
-  return url.replace(/\/$/, "");
+  return url.replace(/\/$/, '')
+}
+
+export function decodeJwt(jwt: string): Jwt {
+  let base64Url = jwt.split('.')[1]
+
+  let pad = base64Url.length % 4
+  if (pad) {
+    if (pad === 1) {
+      throw new Error('Invalid JWT - base64Url is an invalid length to determine padding')
+    }
+
+    base64Url += new Array(5 - pad).join('=')
+  }
+
+  const base64 = base64Url.replace('-', '+').replace('_', '/')
+
+  return JSON.parse(atob(base64))
```

Comment:
It looks like `atob` is only compatible on Node >16 right?

https://developer.mozilla.org/en-US/docs/Web/API/atob#browser_compatibility

Would this need to be polyfilled?